todo_app/data/trello_items.py

```diff
@@ -20,18 +45,19 @@ def get_items():
     Returns: response json() containing trello card names / id's and lists id's
     """
 
-    url = 'https://api.trello.com/1/boards/' + boardid +'/cards?fields=name,idList,dateLastActivity'
+    url = 'https://api.trello.com/1/boards/' + str(get_boardid) +'/cards?fields=name,idList,dateLastActivity'
```

Comment:
Should be
```suggestion
    url = 'https://api.trello.com/1/boards/' + get_boardid() +'/cards?fields=name,idList,dateLastActivity'
```
and similarly in other places - you want the result of calling the function, not a string representation of the function itself.